src/coredump/coredump.c

```diff
@@ -1074,7 +1074,8 @@ static int send_iovec(const struct iovec_wrapper *iovw, int input_fd) {
                         .msg_iov = iovw->iovec + i,
                         .msg_iovlen = 1,
                 };
-                struct iovec copy[2];
+
+                struct iovec copy[2] = {0};
```

Comment:
Note, we use `{}` instead of `{0}`.

Reply:
I saw that but I think Coverity is not happy with it so decided to use `{0}`, I will fix it. Thanks.